app/components/security.js

```diff
@@ -35,10 +36,11 @@ class Security {
             if (securityCookie) {
                 const lostSession = !req.session.expires;
                 const sessionExpired = req.session.expires?req.session.expires <= Date.now():false;
+                const sessionTimeoutCheck = !includes(config.whitelistedPagesIgnoreSessionTimeout, req.originalUrl);
```

Comment:
Do we need lodash for this? Wouldn't `!config.whitelistedPagesIgnoreSessionTimeout.includes(req.originalUrl)` be the same?